### PR TITLE
[webassembly/en] added WebAssembly tutorial

### PR DESCRIPTION
**Pull request by dean-shaff**
- [x] I solemnly swear that this is all original content of which I am the original author
- [x] Pull request title is prepended with `[language/lang-code]`
- [x] Pull request touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Yes, I have double-checked quotes and field names!

### CODE REVIEW

File: wasm.html.markdown [hunk 1]
```diff
@@ -0,0 +1,227 @@
+---
+language: WebAssembly
+filename: learn-wasm.wast
+contributors:
+    - ["Dean Shaff", "http://dean-shaff.github.io"]
+---
+
+```webassembly
```
Comment (divayprakash) [inline wasm.html.markdown]: Please fix this to only ``` ``` ```.

**Comment by divayprakash** [comment]
Closes #3002

**Comment by divayprakash** [comment]
@dean-shaff Thank you for the contribution and welcome to the LXIYM community!

**Comment by jiyinyiyong** [comment]
question, how can we tell the tutorial is WAT or WAST? They are different today.

**Pull request merged**